javascript_nodejs/5.multiturn-prompts-bot/dialogs/mainDialog/index.js

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { ActionTypes, MessageFactory } = require('botbuilder');
+
+const { TextPrompt, NumberPrompt, DialogSet, WaterfallDialog } = require('botbuilder-dialogs');
+
+const DIALOG_STATE_PROPERTY = 'dialogState';
```

Comment:
>DIALOG_STATE_PROPERTY [](start = 6, length = 21)

Why is this one "Property" but the 2 below are "prop" ?

Reply:
standardizing on PROPERT